MonoGame.Framework/Input/GamePad.Android.cs

```diff
@@ -256,19 +320,26 @@ internal static bool OnGenericMotionEvent(MotionEvent e)
             if (e.Action != MotionEventActions.Move)
                 return false;
 
+            if (gamePad._hasCustomMapping)
+            {
+                gamePad.ProcessMotionEvent(e);
```

Comment:
why not just update the code below to use the Axis variables you introduced ?

Reply:
because the code below is faster as it deals with constants. Theorethically the whole performance-wise cost of the fix, for gamepads that didn't need the fix, is a string comparison at creation of the pad and a boolean check per connected pad every frame.